custom_components/contact_energy/sensor.py

```diff
@@ -292,7 +270,6 @@ def update(self):
                 self._state_attributes['previous_day'] = yesterdaysData
         else:
             _LOGGER.error('Unable to log in!')
-
     def make_attribute(self, response, value):
```

Comment:
Why is make_attribute defined twice?

Reply:
See my above comment, I messed up adding back in the commented out code.